packages/frontend/store/system/reducers.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  GET_WIDTH,
```

Comment:
Shouldn't these all be SET_WIDTH? Not GET?

Reply:
I named it GET_WIDTH because I'm getting window.innerWidth

Reply:
Isn't your reducer **setting** the width??

Reply:
The reducer isn't actually setting the width. The width changes when the user resizes the window and I'm **getting** the width. As opposed to setTheme where I'm **setting** the theme with.